README.md

```diff
@@ -4,27 +4,55 @@
 
 This proposal is at stage 2 of [the TC39 process](https://tc39.es/process-document/).
 
+## Terminology
+
+This proposal initially used the term “sequence properties”, but that is a misnomer.
+A sequence of characters is a string, and a string property is one whose *values* (the codomain) are strings,
+just like a binary property is one whose values are binary true/false
+(that is, whether the property applies or does not apply).
+
+Unicode has since formalized this,
+using “property of code points” vs. “property of strings” for the *domain* of a property.
+See https://www.unicode.org/reports/tr18/#domain_of_properties
+
+Also, we mostly use “character” and “code point” interchangeably.
+More formally, “character” refers to assigned code points, but properties have values for all code points.
+(Most properties map all unassigned code points to one default value.)
+
 ## Motivation
 
-The Unicode Standard assigns various properties and property values to every symbol. For example, to get the set of symbols that are used exclusively in the Greek script, search the Unicode database for symbols whose `Script` property is set to `Greek`.
+The Unicode Standard assigns various properties and property values to every character/code point.
+For example, to get the set of characters that are used exclusively in the Greek script,
+search the Unicode database for characters whose `Script` property value is `Greek`.
```

Comment:
the Unicode database provides for determining exactly those characters whose `Script` property value is `Greek`.

Reply:
done